actors/common/src/main/java/org/sunbird/common/ShadowUserProcessor.java

```diff
@@ -129,7 +133,8 @@ private void updateStatusInUserOrg(ShadowUser shadowUser, String id) {
     }
 
     private void updateUser(ShadowUser shadowUser) {
-            List<Map<String, Object>> esUser = getUserMatchedIdentifierFromES(shadowUser);
+        ExecutorService executor = Executors.newFixedThreadPool(N_THREADS);
```

Comment:
define ExecutorService out side other wise for all 20k it has to create new pool
* check once job is completed , close executor
* this method can take Executor ref

Reply:
done